docker/Dockerfile.tcf-dev

```diff
@@ -111,17 +142,15 @@ RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.7.101.3.bin \
 
 
 WORKDIR /tmp
-# Build ("Untrusted") OpenSSL, then build ("trusted") SGX OpenSSL
-# Note: The latter will compile in HW or SIM mode depending on the
+ENV OPENSSL_VER=1.1.1d
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER /tmp/openssl-$OPENSSL_VER/
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER.tar.gz /tmp/
+
+# Build ("trusted") SGX OpenSSL
+# Note: This will compile in HW or SIM mode depending on the
 # availability of /dev/isgx and /var/run/aesmd/aesm.socket
-RUN OPENSSL_VER=1.1.1d \
- && wget https://www.openssl.org/source/openssl-$OPENSSL_VER.tar.gz \
- && tar -zxf openssl-$OPENSSL_VER.tar.gz \
- && cd openssl-$OPENSSL_VER/ \
- && ./config \
+RUN cd openssl-$OPENSSL_VER/ \
```

Comment:
Does building SGXSSL using command `make SGX_MODE=${SGX_MODE} DESTDIR=/opt/intel/sgxssl all test' rebuild openSSL again ?

Reply:
Yes, it builds openSSL again.